### DISCUSSION TITLE
ClearRefinement on  two widgets with the same attribut

### DISCUSSION THREAD

**Discussion by fvsadem** [algolia/instantsearch#4482] [General] [answered]
Hi,
 
I have two widgets for the same attribut "keywords". One widget is used to filter some keywords and the other is just for a specific keyword.

To my purpose, i have to use the operator "and" on one widget.

But when i clic on "clearRefinement" button, it doesn't works (the two widgets are not clear).
When i use operator "or" it works perfectly.

Do i miss something ?

Thanks

**Comment by Haroenv** [comment, marked as answer]
It's possible that this is not behaving as we expect, since IIRC `clearRefinement` might use the attribute as a key and find only the first face type for this attribute. Do you have a sandbox with this behaviour @fvsadem ? We have a template available for that purpose [here](https://codesandbox.io/s/github/algolia/create-instantsearch-app/tree/templates/instantsearch.js). Thanks!

**Reply by fvsadem** [reply]
Thanks @Haroenv ,

Sandbox : https://codesandbox.io/s/practical-snow-u0pjc

I just have tested that if i use "and" operator on the two widgets, it's working !

For my purpose, i will leave "and" on both.

Bye
